mineral-ui/mineral-ui
# [Site] Getting Started

connects #269 

### Description
- Add real content to Getting Started page
- Add CodeBlock component

### Motivation and context

People need to know how to get started, even if they don’t visit out GitHub repo.

### Screenshots or videos, if appropriate


### How to test


1. `cd mineral-ui/packages/site && npm start`
2. It should redirect you to /getting-started (but that's an issue with #275 if it doesn’t work), if not, navigate there
3. Read the content

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Get started](https://media.giphy.com/media/3oz8xXrA14O8P0DzW0/giphy.gif)
